atlasdb-config/src/main/java/com/palantir/atlasdb/factory/TransactionManagers.java

```diff
@@ -453,22 +451,22 @@ private static PersistentLockService createAndRegisterPersistentLockService(
      * @deprecated Not intended for public use outside of the AtlasDB CLIs
      */
     @Deprecated
-    public static LockAndTimestampServices createLockAndTimestampServices(
+    public static LockAndTimestampServices createLockAndTimestampServicesForCli(
             AtlasDbConfig config,
+            java.util.function.Supplier<AtlasDbRuntimeConfig> runtimeConfigSupplier,
             Consumer<Object> env,
             com.google.common.base.Supplier<LockService> lock,
-            com.google.common.base.Supplier<TimestampService> time) {
+            com.google.common.base.Supplier<TimestampService> time,
+            TimestampStoreInvalidator invalidator,
+            String userAgent) {
```

Comment:
It's a shame we didn't upgrade Guava to 21! (Guava Supplier becomes a Java Supplier in 21)